test/experimental/isomorphism.jl

```diff
@@ -79,57 +79,61 @@ end
     # TODO better tests for vertex_relation and edge_relation
     vrel(u, v) = false
     erel(e1, e2) = false
-    @test has_isomorph(complete_graph(4), complete_graph(4), vertex_relation=vrel) == false
-    @test has_isomorph(complete_graph(4), complete_graph(4), edge_relation=erel) == false
-    @test has_subgraphisomorph(complete_graph(4), complete_graph(3), vertex_relation=vrel) == false
-    @test has_subgraphisomorph(complete_graph(4), complete_graph(3), edge_relation=erel) == false
-    @test has_induced_subgraphisomorph(complete_graph(4), complete_graph(3), vertex_relation=vrel) == false
-    @test has_induced_subgraphisomorph(complete_graph(4), complete_graph(3), edge_relation=erel) == false
-
-    @test count_isomorph(complete_graph(4), complete_graph(4), vertex_relation=vrel) == 0
-    @test count_isomorph(complete_graph(4), complete_graph(4), edge_relation=erel) == 0
-    @test count_subgraphisomorph(complete_graph(4), complete_graph(3), vertex_relation=vrel) == 0
-    @test count_subgraphisomorph(complete_graph(4), complete_graph(3), edge_relation=erel) == 0
-    @test count_induced_subgraphisomorph(complete_graph(4), complete_graph(3), vertex_relation=vrel) == 0
-    @test count_induced_subgraphisomorph(complete_graph(4), complete_graph(3), edge_relation=erel) == 0
-
-    @test isempty(all_isomorph(complete_graph(4), complete_graph(4), vertex_relation=vrel))
-    @test isempty(all_isomorph(complete_graph(4), complete_graph(4), edge_relation=erel))
-    @test isempty(all_subgraphisomorph(complete_graph(4), complete_graph(3), vertex_relation=vrel))
-    @test isempty(all_subgraphisomorph(complete_graph(4), complete_graph(3), edge_relation=erel))
-    @test isempty(all_induced_subgraphisomorph(complete_graph(4), complete_graph(3), vertex_relation=vrel))
-    @test isempty(all_induced_subgraphisomorph(complete_graph(4), complete_graph(3), edge_relation=erel))
-
-    # some test for early returns if there is no isomorphism
-    @test count_isomorph(complete_graph(4), cycle_graph(4)) == 0
-    @test isempty(all_isomorph(complete_graph(4), cycle_graph(4)))
-    @test count_subgraphisomorph(complete_graph(3), complete_graph(4)) == 0
-
-
-    # this tests triggers the shortcut in the vf2 algorithm if the first graph is smaller than the second one
-    @test has_isomorph(complete_graph(3), complete_graph(4)) == false
+    @testset "complete graph" begin
```

Comment:
This is minor - but while you're in here could you fix the indentation to be 4 spaces?